modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * metering/configuring-metering/metering-configure-reporting-operator.adoc
+
+[id="metering-exposing-the-reporting-api_{context}"]
+= Exposing the reporting API
+
+On {poduct-title} the default metering installation automatically exposes a Route making the reporting API available. This provides the following features:
+
+* Automatic DNS
+* Automatic TLS based on the cluster CA
+
+As well, the default installation makes it possible to take advantage of the Openshift service serving certificates to protect the reporting API with TLS. The Openshift OAuth proxy is deployed as a side-car container for `reporting-operator`, which protects the reporting API with authentication.
```

Comment:
s/As well/Also,
s/take advantage of/use
s/Openshift/OpenShift  (or {product-title} ) -- in a few spots